apps/aecore/src/aec_governance.erl

```diff
@@ -22,8 +22,12 @@ blocks_to_check_difficulty_count() ->
     ?BLOCKS_TO_CHECK_DIFFICULTY_COUNT.
 
 expected_block_mine_rate() ->
-    application:get_env(aecore, expected_mine_rate,
-                        ?EXPECTED_BLOCK_MINE_RATE).
+    case aeu_env:user_config([<<"mining">>, <<"expected_mine_rate">>]) of
```

Comment:
This would be equivalent to
```erlang
aeu_env:user_config_or_env([<<"mining">>, <<"expected_mine_rate">>],
                            aecore, expected_mine_rate,
                            ?EXPECTED_BLOCK_MINE_RATE)
```

Reply:
Already changed locally, will push soon

Reply:
in 0767c438